docs/validators/sigstore_cosign.md

```diff
@@ -349,4 +355,37 @@ To disable this feature the `with.verifyInTransparencyLog` key can be set to `fa
 
 ### Keyless signatures
 
-Keyless signatures have not yet been implemented but are planned in upcoming releases.
+Keyless signatures are a feature of sigstore that allows to sign container images without the need to manage a private key. Instead the signatures are bound to identities, attested by OIDC providers, and use ephemeral keys, short-lived certificates and a transparency log under the hood to provide the same security guarantees. Further information on this topic can be found [here](https://docs.sigstore.dev/signing/overview/).
+
+When using keyless signatures, the `trustRoots[*].keyless` field can be used to specify the issuer and subject of the keyless signature. The issuer is the OIDC provider that attests the identity and the subject is the identity that created the keyless signature. Both fields are regexes, so it is possible to use the full URL of the OIDC provider or only parts of it. The subject is usually an email address, but can also be only parts of it. The following example shows how to configure a validator for keyless signatures:
+
+```yaml title="charts/connaisseur/values.yaml"
+- name: keylessvalidator
+  type: cosign
+  trustRoots:
+  - name: keyless
+    keyless:
+      issuer: "github"
+      subject: "philipp.belitz@securesystems.de
+```
+
+In case the signature was created using the sigstore infrastructure, nothing else needs to be configured since Connaisseur will automatically retrieve all needed public keys and certificates from the sigstore infrastructure. If the signature was created using a private infrastructure, the `host.fulcioCert` field can be used to specify the root certificate belonging to the fulcio CA which was used to create the keyless signatures. The `host.fulcioCert` field should contain the root certificate in PEM format. The same applies to the `host.ctLogPubkey` field which can be used to specify the public keys needed for verifying Signed Certificate Timestamps (SCT) and the `host.rekorPubkey` field which can be used to specify the public key used to verify the signature of log entries from Rekor.
```

Comment:
```suggestion
In case the signature was created using the Sigstore infrastructure, nothing else needs to be configured since Connaisseur will automatically retrieve all needed public keys and certificates from the Sigstore infrastructure. If the signature was created using a private infrastructure, the `host.fulcioCert` field can be used to specify the root certificate belonging to the Fulcio CA which was used to create the keyless signatures. The `host.fulcioCert` field should contain the root certificate in PEM format. The same applies to the `host.ctLogPubkey` field which can be used to specify the public keys needed for verifying Signed Certificate Timestamps (SCT) and the `host.rekorPubkey` field which can be used to specify the public key used to verify the signature of log entries from Rekor.
```